app/models/group.rb

```diff
@@ -0,0 +1,8 @@
+class Group < ApplicationRecord
+  validates :name,
+            :responsable_name,
+            :responsable_contact,
+            :from_date,
+            :to_date,
+            presence: true
+end
```

Comment:
@elle I'm sorry I rebased and pushed and your previous comment on this one disappeared!
By the way this way is working! :)

Reply:
👍 

I would just indent with two spaces, not aligned to `:name`

Reply:
ok!